Sources/Rendering/OpenGL/Texture/index.d.ts

```diff
@@ -0,0 +1,316 @@
+import {Wrap, Filter} from "./Constants";
+import vtkOpenGLRenderWindow from '../RenderWindow';
+import {Nullable} from '../../../types';
+import { vtkAlgorithm, vtkObject } from "../../../interfaces";
+import { VtkDataTypes } from "../../../Common/Core/DataArray";
+
+/**
+ * Represents the time with a modification timestamp.
+ */
+interface Time{
+  mtime:number;
+}
+
+/**
+ * Initial values for creating a new instance of vtkOpenGLTexture.
+ */
+export interface ITextureInitialValues {
+  _openGLRenderWindow: Nullable<vtkOpenGLRenderWindow>;
+  _forceInternalFormat: boolean;
+  context?: WebGLRenderingContext | WebGL2RenderingContext;
+  handle: number;
+  sendParametersTime: Time;
+  textureBuildTime: Time;
```

Comment:
These should be typed as `vtkObject`, since `macro.obj` creates a base VTK object.